brainiak/utils/fmrisim.py

```diff
@@ -752,41 +766,41 @@ def _double_gamma_hrf(response_delay=6,
 def convolve_hrf(stimfunction,
                  tr_duration,
                  hrf_type='double_gamma',
-                 scale_function=1,
+                 scale_function=True,
                  temporal_resolution=1000.0,
                  ):
     """ Convolve the specified hrf with the timecourse
 
-        Parameters
-        ----------
+    Parameters
+    ----------
 
-        stimfunction : timepoint by timecourse array
-            What is the time course of events to be modelled in this
-            experiment. This can specify one or more timecourses of events.
-            The events can be weighted or binary
+    stimfunction : timepoint by timecourse array
+        What is the time course of events to be modelled in this
+        experiment. This can specify one or more timecourses of events.
+        The events can be weighted or binary
 
-        tr_duration : float
-            How long (in s) between each volume onset
+    tr_duration : float
+        How long (in s) between each volume onset
 
-        hrf_type : str or list
-            Takes in a string describing the hrf that ought to be created.
-            Can instead take in a vector describing the HRF as it was
-            specified by any function
+    hrf_type : str or list
```

Comment:
I remember there may be some requirement of brainiak code formatting that default values should be explicitly documented. But for this case, I suggest at least documenting the possible values that can be accepted. For example, which strings are acceptable?

Reply:
Added some details